Karma/Controllers/PostsController.cs

```diff
@@ -7,21 +7,28 @@
 using Microsoft.EntityFrameworkCore;
 using Karma.Data;
 using Karma.Models;
+using Microsoft.AspNetCore.Http;
+using System.IO;
+using Microsoft.AspNetCore.Hosting;
 
 namespace Karma.Controllers
 {
     public class PostsController : Controller
     {
         private readonly KarmaContext _context;
 
-        public PostsController(KarmaContext context)
+        private readonly IWebHostEnvironment _iWebHostEnv;
```

Comment:
What is it? it lacks comments

Reply:
Added a short explanation in the following commit.